includes/class-payment-information.php

```diff
@@ -264,6 +275,26 @@ public static function get_token_from_request( array $request ) {
 		return $token;
 	}
 
+	/**
+	 * Extract the payment CVC confirmation from the provided request.
+	 *
+	 * @param array $request Associative array containing payment request information.
+	 *
+	 * @return string|NULL
+	 */
+	public static function get_cvc_confirmation_from_request( array $request ) {
+		$payment_method    = $request['payment_method'] ?? null;
+		$token_request_key = 'wc-' . $payment_method . '-payment-cvc-confirmation';
+		if (
+			! isset( $request[ $token_request_key ] ) ||
```

Comment:
Is there supposed to be a `"wc-woocommerce_payments-payment-cvc-confirmation"` key in the request data? I'm not seeing that in the request data, so this is always turning out to be `null` for me. Is there some flow that I should test where this will have a value?

Reply:
I'm working to send this variable on Platform Checkout to validate the CVC number.